indico/modules/events/timetable/models/breaks.py

```diff
@@ -81,6 +82,10 @@ def end_dt(self):
     def __repr__(self):
         return format_repr(self, 'id', _text=self.title)
 
+    @locator_property
+    def locator(self):
+        return dict(self.timetable_entry.locator, break_id=self.id)
```

Comment:
`self.timetable_entry.event_new.locator` - you don't want the timetable entry id in this locator